css/site.styl

```diff
@@ -124,6 +118,18 @@ img
 
 // ******************************************************
 
+// FOOTER
+
+#footer
+  line-height: 1
```

Comment:
no because the type has its own descender and line gap, whereas the svg doesnt. so in order to get them to be pixel perfect centered i had to do this. if you want to do it another way go ahead